ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -58,97 +53,120 @@ class FaultInjector
     NoForceData = 8
     };
 
-	static constexpr std::bitset<10> isSystem{		0b00'0000'0001 };
-	static constexpr std::bitset<10> isArmGoalError{		0b00'0000'0010 };
-	static constexpr std::bitset<10> isArmExecutionError{		0b00'0000'0100 };
-	static constexpr std::bitset<10> isTaskGoalError{	0b00'000'1000 };
-	static constexpr std::bitset<10> isCamGoalError{	0b00'0001'0000 };
-	static constexpr std::bitset<10> isCamExecutionError{	0b00'0010'0000 };
-	static constexpr std::bitset<10> isPanTiltGoalError{		0b00'0100'0000 };
+  //system
+	static constexpr std::bitset<10> isSystem{		            0b00'0000'0001 };
+	static constexpr std::bitset<10> isArmGoalError{          0b00'0000'0010 };
+	static constexpr std::bitset<10> isArmExecutionError{     0b00'0000'0100 };
+	static constexpr std::bitset<10> isTaskGoalError{	        0b00'0000'1000 };
+	static constexpr std::bitset<10> isCamGoalError{	        0b00'0001'0000 };
+	static constexpr std::bitset<10> isCamExecutionError{	    0b00'0010'0000 };
+	static constexpr std::bitset<10> isPanTiltGoalError{	  	0b00'0100'0000 };
 	static constexpr std::bitset<10> isPanTiltExecutionError{	0b00'1000'0000 };
 	static constexpr std::bitset<10> isLanderExecutionError{	0b01'0000'0000 };
-	static constexpr std::bitset<10> isPowerSystemFault{	0b10'0000'0000 };
-  
-private:
-  float powerTemperatureOverloadValue;
+	static constexpr std::bitset<10> isPowerSystemFault{    	0b10'0000'0000 };
   
+  //power
+  static constexpr std::bitset<3> islowVoltageError{ 0b001 };
+	static constexpr std::bitset<3> isCapLossError{  	 0b010 };
+	static constexpr std::bitset<3> isThermalError{	   0b100 };
```

Comment:
The reason these lines appear unaligned on GitHub because you are mixing tabs and spaces. Convert all tabs to spaces.